frontend/packages/component-library-styles/CONTRIBUTING.md

```diff
@@ -0,0 +1,153 @@
+✅ Use **semantic colors** from `@code-dot-org/component-library-styles/colors.scss` where possible.  
+✅ Keep token names **descriptive** and follow existing naming conventions.  
+✅ Keep file size reasonable — split large files if needed.
+
+## 🧪 How to Test Your Styles
+
+There's currently no automated testing for styles. Here's how you can test your changes manually or through other
+packages automated tests:
+
+1. **Unit Tests(Manually)** — Test RTL(right-to-left) support directly since it's part of Design System scope.
+2. **Storybook Tests** — Ensure Storybook displays the token or style correctly.
+3. **Accessibility Tests** — Run Axe tests to verify accessibility in light and dark themes.
+4. **Visual Regression Tests** — Ensure the style appears correctly across browsers and devices.
+
+➡️ See the full [Testing Guide](./README.md#testing) in the README.
+
+## 🔀 Commit Message Guidelines
+
+We follow the Conventional Commits format for all commit messages to ensure a consistent and readable git history.
+
+```bash
+<type>(<scope>): <description>
+```
+
+### ✅ Types:
+
+| Type       | Description                                            |
+| ---------- | ------------------------------------------------------ |
+| `feat`     | A new feature                                          |
+| `fix`      | A bug fix                                              |
+| `docs`     | Documentation updates                                  |
+| `refactor` | Code changes that neither fix a bug nor add a feature  |
+| `test`     | Adding missing tests or updating existing tests        |
+| `chore`    | Maintenance tasks (build process, tools, dependencies) |
+| `perf`     | Performance improvements                               |
+
+### ✅ Examples:
+
+```bash
+feat(colors): add new semantic token for primary background
+
+fix(mixins): correct font size in button-text-large
+
+docs(typography): update documentation for heading mixins
+```
+
+### ✅ Best Practices:
+
+- Use present tense (“add” not “added”).
+- Use lowercase for the type and scope.
+- Keep the description concise and specific.
+
+## 📚 Need Help?
+
+If you need help at any point, reach out to:
+
+- `#ask-design-system` Slack channel
+- @moshebaricdo, @markabarrett for design questions
+- @levadadenys for technical implementation questions
+
+## 💡 Tips for a Smooth PR Review
+
+✅ Keep the PR focused on **one token/group of tokens or style**.  
+✅ Add **clear, descriptive commit messages**.  
+✅ Add screenshots or recordings to demonstrate UI changes.  
+✅ Keep PRs under **500 lines** where possible (if larger, split into smaller PRs).
+
+✅ For larger PRs, consider creating a Draft PR for early feedback.
+
+## ✅ How to Get Your PR Approved Faster
+
+- If there’s no response after **48 hours**, politely ping the reviewer in Slack.
+- If you receive feedback, **respond within 24–48 hours** to keep the momentum going.
```

Comment:
Just calling out that these feel a bit long. I'd normally expect first responses within 24 hours. I know timezones make things harder, but still.

I might keep the first point (after 48 hours, feel free to ping people) but suggest responding to feedback within 24 hours.